text/0034-telemetry-policies.md

```diff
@@ -0,0 +1,146 @@
+
+We will keep telemetry enabled by default while leaving the option for users to disable it. See Impact on Documentation part.
+
+> Some new features will have telemetry data points to check the assumptions and measure objectives achievements.
+
+Each new feature specification should list data points to be collected and measured. Obviously the specification must explain why these data points are necessary for relation to the feature.
+
+We will simplify the deactivation of all our telemetries in one action. To do that, we will remove the Sentry specific flag `MEILI_NO_SENTRY` to merge its use into the `MEILI_NO_ANALYTICS` flag. This means that its activation and deactivation will be shared with analytics within the `MEILI_NO_ANALYTICS` flag. By default, Sentry metrics will stay activated.
+
+#### Current stack for analytics
+
+We are currently using Amplitude as a quantitative analytic tool to measure usage as a big picture.
+
+E.g.
+- Cohort: Servers that are running for more than seven days with the old version of MS (< v0.10.1)
+- Cohort: Servers that are running for more than seven days with the new version of MS (> v0.10.1)
+- Cohort: Servers that are running for more than 30 days with the old version of MS (< v0.10.1)
+- Cohort: Servers that are running for more than 30 days with the new version of MS (> v0.10.1)
+- Graph: The total of every telemetry messages received
+- Graph: The total of every unique id seen
+- Graph: The number of servers that are running at least since 7 days
+- Graph: The number of servers that are running at least since 30 days
+- User Composition: Device composition
+- User Composition: Version composition
+- Indexes size distribution for more than seven days (<200k not taken into account)
+- Indexes size distribution for the others
+
+#### Future Metrics
+
+##### Number of indexes per DB
+Discovery metric for federated search feature.
+
+##### Number of documents per indexes
+Discovery metric to know the average number of documents are stored in an index.
+
+##### Geographical server distribution, CPU server distribution, RAM server distribution, Disk server distribution
+Discovery metrics to choose more relevant data centers over the world for the SaaS cloud platform. Will also be used in order to perform price studies analysis.
+
+For the Geographical Server Distribution we need to make a call to a third-party like http://ip-api.com/json/113.14.168.85 to get `Country`, `City` and `Provider` at MeiliSearch launch.
+
+Staying transparent is important to us, which is why the metrics will be explained in details on a change log, blog post and a dedicated documentation page.
+
+For the CPU, RAM and DISK metrics, we can use an internal system stat method to get this information.
+
+##### SDK distribution
+Discovery metric to check adoption of our new sdks and the use of existing ones
```

Comment:
it will be SDK's job to add this information in the header?

Reply:
Yes, in the `User-Agent` header